dev-packages/application-package/src/application-props.ts

```diff
@@ -119,6 +119,15 @@ export interface FrontendApplicationConfig extends ApplicationConfig {
      */
     readonly disallowReloadKeybinding?: boolean;
 
+    /**
```

Comment:
I don't like that we put these properties on top-level, although they specific only for electron. Maybe introduce a subsection. I think `disallowReloadKeybinding` should be deprecated and moved there as well.

Reply:
Will take care 👍

Reply:
Putting `disallowReloadKeybinding` into it's own `electron` property should be done by another PR. I've changed this PR and this change is out of scope for this PR.